SoftwareName_Adapter/SoftwareName_Adapter.csproj

```diff
@@ -0,0 +1,46 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
While this ofc is nice if it does work, wonder if we should default to framework 4.7.2 here (at least for the adapter) as a lot of them requires you to be on framework for it to work.

@FraserGreenroyd good to get your input here as well though.

Reply:
To be clear, using the new csproj format, but pointing to the standard framework.

Reply:
@FraserGreenroyd what do you think?

Reply:
Sorry been caught up, just catching up on this.

Personally I prefer defaulting to `netstandard2.0` as our long term vision is the upgrading. `netstandard2.0` supports both framework and core as it bridges the two. So adapters should not have any issue with using `netstandard2.0`, and if any did, whoever was developing them ought to be skilled enough to downgrade the project file if it was really needed, so personally I'd prefer we go to `netstandard2.0` so we can begin the process of upgrading things by starting with new things 😄

Reply:
@IsakNaslundBh due to this being needed urgently I think we could merge this and then raise an issue if we change our mind on the target framework. @al-fisher